### ISSUE TITLE
Error occurs if the GitHub App used by Renovate is Suspended by an owner

### ISSUE THREAD

**Issue by miyajan** [mend/renovate-ce-ee#391]
Renovate Community Edition 6.5.0

If an owner suspends Renovate's GitHub App, the following error occurs.

```
DEBUG: Performing app sync
DEBUG: POST https://<GITHUB_ENDPOINT>/app/installations/48/access_tokens = (code=ERR_NON_2XX_3XX_RESPONSE, statusCode=403 retryCount=0, duration=52)
ERROR: Error fetching installation access token
       "err": {
         "name": "HTTPError",
         "code": "ERR_NON_2XX_3XX_RESPONSE",
         ...
         "message": "Response code 403 (Forbidden)",
         "stack": "HTTPError: Response code 403 (Forbidden)\n    at Request.<anonymous> (/usr/src/app/node_modules/got/dist/source/as-promise/index.js:118:42)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)",
         "options": {
           "headers": {
             "user-agent": "RenovateBot/37.33.3 (https://github.com/renovatebot/renovate)",
             "accept": "application/vnd.github+json, application/vnd.github.v3+json",
             "authorization": "***********",
             "x-github-api-version": "2022-11-28",
             "accept-encoding": "gzip, deflate, br"
           },
           "url": "https://<GITHUB_ENDPOINT>/app/installations/48/access_tokens",
           "hostType": "github",
           "username": "",
           "password": "",
           "method": "POST",
           "http2": false
         },
         "response": {
           "statusCode": 403,
           "statusMessage": "Forbidden",
           "body": {
             "message": "This installation has been suspended",
             "documentation_url": "https://docs.github.com/enterprise-server@3.8/rest/reference/apps#create-an-installation-access-token-for-an-app"
           },
           ...
         }
       }
 WARN: Failed retrieving installation access token - abort fetching installation repositories
       "installation": {"installationId": <INSTALLATION_ID>, "organizationName": "<OWNER_NAME>"}
 WARN: Failure getting all app installations repositories - abort
       "installation": {
         ...
        }
 WARN: Could not retrieve app installations - abort sync
DEBUG: App sync completed
```

Renovate 5.1.0 did not generate any errors and was working. It was broken in Renovate 6.

**Comment by Gabriel-Ladzaretti** [comment]
Thank you for the bug report. We will address this as soon as possible. In the meantime, if possible, please uninstall the org instead of suspending it. Apologies for the inconvenience.

**Comment by Gabriel-Ladzaretti** [comment]
Addressed in `v6.7.0`.
